config/DoctrineClassMapping/WMDE.Fundraising.PaymentContext.Domain.Model.PaymentReferenceCode.dcm.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping https://www.doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+    <entity name="WMDE\Fundraising\PaymentContext\Domain\Model\PaymentReferenceCode" table="payment_reference_codes">
+        <unique-constraints>
+            <unique-constraint name="code" columns="code"/>
+        </unique-constraints>
+        <id name="id" type="integer" column="id">
+            <generator strategy="IDENTITY"/>
+        </id>
+        <field name="formattedCode" type="string" column="code" nullable="true"/>
```

Comment:
This should not be nullable, now that payment reference codes are their own instance

Now that it's a string, you can also delete the PaymentReferenceCode doctrine mapping class and its tests and initialization.